.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,17 @@
+blank_issues_enabled: false
```

Comment:
Does this do anything if we don't have an issue template? Maybe better to remove?

```suggestion
```

Reply:
Not 100% sure, but it disables the creation of blank issues (without a template). Without a template, I guess it would disable issue creation (it hides the option from the chooser).

Alternatively, you could create an issue template (form) that has a title and a markdown (with no fields), repeating the stuff below again.

Reply:
Hm, no need for an issue template right now, so removing the option. Thanks.

Reply:
Seems it requires

```
blank_issues_enabled: true
```

:sweat_smile: